src/background/adblocker.js

```diff
@@ -311,36 +303,38 @@ async function injectCosmetics(msg, sender) {
 
 chrome.runtime.onMessage.addListener((msg, sender) => {
   if (msg.action === 'getCosmeticsFilters') {
-    injectCosmetics(msg, sender).catch((e) =>
-      console.error(
-        `[adblocker] Error while processing cosmetics filters: ${e}`,
-      ),
-    );
+    injectCosmetics(msg, sender);
   }
 
   return false;
```

Comment:
Only a detail since it has no changed. But in principle, we could also remove the "return false" here, couldn't we? The default of returning undefined, should have the same effect.